### PR TITLE
shiba, husky, akita: initial crDroid 11

### PR DESCRIPTION
**Pull request by iogh07**

### CODE REVIEW

File: akita.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-15-shiba-husky-akita-crdroid-11-0-beta-official-01-11-2024.4650502",
+			"gapps": "https://s.go.ro/oigts9p2",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/akita/11.x/recovery",
+			"paypal": "https://paypal.me/igherman96",
+			"telegram": "https://t.me/ionutbuilds",
+			"dt": "https://github.com/crdroidandroid/android_device_google_akita",
```
Comment (gwolf2u) [inline akita.json:23]: missing 15.0 branch

File: akita.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_zuma",
```
Comment (gwolf2u) [inline akita.json:24]: same here

File: husky.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-15-shiba-husky-akita-crdroid-11-0-beta-official-01-11-2024.4650502",
+			"gapps": "https://s.go.ro/oigts9p2",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/husky/11.x/recovery",
+			"paypal": "https://paypal.me/igherman96",
+			"telegram": "https://t.me/ionutbuilds",
+			"dt": "https://github.com/crdroidandroid/android_device_google_husky",
```
Comment (gwolf2u) [inline husky.json]: missing repo

File: husky.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_zuma",
```
Comment (gwolf2u) [inline husky.json:24]: 15.0 ?

File: shiba.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-15-shiba-husky-akita-crdroid-11-0-beta-official-01-11-2024.4650502",
+			"gapps": "https://s.go.ro/oigts9p2",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/shiba/11.x/recovery",
+			"paypal": "https://paypal.me/igherman96",
+			"telegram": "https://t.me/ionutbuilds",
+			"dt": "https://github.com/crdroidandroid/android_device_google_shiba",
```
Comment (gwolf2u) [inline shiba.json]: missing repo

File: shiba.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_zuma",
```
Comment (gwolf2u) [inline shiba.json:24]: 15.0 ?

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: shiba.json (hunk 5 above)
Comment (iogh07) [reply shiba.json]: done

**Review by gwolf2u — APPROVED** [review]

**Comment by Smiley-McSmiles** [comment]
Flaahed my Pixel 8a (akita).
Couple issues so far:
1. Notifications are boxy not rounded
2. Default launcher crashes when long pressing firefox web apps (Add to homescreen) (switched to lawnchair) 

Sidenote: would love for the built in tethering options to bypass carrier restrictions. (Currently having to use the tetherfi app which uses a proxy setup to achieve this) 
Sorry I'm posting this here. I couldn't easily find a bug reporting section.

**Pull request merged**